README.md

```diff
@@ -2,6 +2,9 @@ Title: endless-runner-html5-game<br>
 Author: Steven Lambert<br>
 Website: http://blog.sklambert.com/endless-runner-html5-game/<br>
 Play Game: https://tutsplus.github.io/kandi-runner/<br>
+
+[![CarDEalerShip](https://raw.githubusercontent.com/ashishsiot/endless-runner-html5-game/master/game.PNG)](https://tutsplus.github.io/kandi-runner/)
```

Comment:
What's with the `CarDEalerShip` image alt? Also, could you use this repos as the image (and not yours) or just paste the image into the file (github will resolve it and host it themselves).

Reply:
I Make One Note Of All Syntax Of .md (Markdown Language )  And Whenever I Want To Make Changes In Any Ones Repo I Will Just Simply Copy Past The Syntax And Make Changes But This Time I Will Forget To Change Header Name.

And The About Image Address Will Just Upload Image Into My Fork Repo And Use This Hosted Image Address.   

- I Am Really Sorry About That I Am Still Learning And Thank You For Showing My Mistakes I Hope You Will Help Me To Grow Up 

                                                                                                                                                                         -Thank You